src/posthog-featureflags.js

```diff
@@ -161,9 +161,9 @@ export class PostHogFeatureFlags {
             for (let i = 0; i < flags.length; i++) {
                 flagsObj[flags[i]] = true
             }
-            this.instance.persistence.register({ '$override_feature_flags': flagsObj })
+            this.instance.persistence.register({ $override_feature_flags: flagsObj })
         } else {
-            this.instance.persistence.register({ '$override_feature_flags': flags })
+            this.instance.persistence.register({ $override_feature_flags: flags })
```

Comment:
This change is coming from eslint on the post commit hook. I think it was introduced here: https://github.com/PostHog/posthog-js/pull/284/files